.github/workflows/report_a_quiz.yaml

```diff
@@ -14,7 +16,55 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@master
-      - name: Send Quiz message to the channel
+      - name: Config the environment variable
         shell: bash
         run: |
-          ./report_quiz.sh ${{ secrets.TELEGRAM_OSSCAMEROON_GROUP_ID }} ${{ secrets.TELEGRAM_BOT_TOKEN }} ${{ secrets.QUIZAPI_KEY }}
+          echo "TELEGRAM_BOT_TOKEN=${{ secrets.TELEGRAM_BOT_TOKEN }}" >> env.sh
+          echo "TELEGRAM_CHAT_ID=${{ secrets.TELEGRAM_OSSCAMEROON_GROUP_ID }}" >> env.sh
+          echo "QUIZAPI_KEY=${{ secrets.QUIZAPI_KEY }}" >> env.sh
+      
+      - name: Load the database
+        run: |
+            git init
+            git remote add origin https://${{ secrets.REPORTQUIZ_DATA_REPO_TOKEN }}@github.com/osscameroon/reportquiz_data.git
+            git pull origin main
+            git switch -c main
+
+      - name: Fetch quizzes user answers
+        shell: bash
+        run: |
+          source main.sh
+          # update the quiz database
+          fetch_quiz_user_answers
+
+      - name: Check the status of the competition
+        shell: bash
+        run: |
+          source main.sh
+          # start a new competition each monday morning
+          if [ $(date +%a_%p) == Mon_AM ]; then start_quiz_competition; fi
+          # end the competition each sunday evening
+          if [ $(date +%a_%p) == Sun_PM ]; then
+            stop_quiz_competition
+            touch /tmp/skip_send_quiz
+          fi
```

Comment:
The competition thing is nice, but is not necessary for this initial version, and I think it makes the bot a little more complicated than it should have been.

Reply:
> it makes the bot a little more complicated than it should have been.

I don't understand